pkg/cmd/extension/command.go

```diff
@@ -420,7 +420,7 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 			var flagForce bool
 			var flagDryRun bool
 			cmd := &cobra.Command{
-				Use:   "upgrade {<name> | --all}",
+				Use:   "upgrade [<name> | --all]",
```

Comment:
Similar to the `gh gist delete` comment above, I'm unsure what the right thing to do here is.